src/GraphQL.php

```diff
@@ -139,21 +143,23 @@ public function executeBatchedQueries(array $requests, GraphQLContext $context,
      * To render the ExecutionResult, you will probably want to call `->toArray($debug)` on it,
      * with $debug being a combination of flags in \GraphQL\Error\Debug
      *
-     * @param  string  $query
+     * @param  string|\GraphQL\Language\AST\DocumentNode  $query
```

Comment:
Or the other way around: `\GraphQL\Language\AST\DocumentNode|string`

Reply:
It's a string most of the time, and the base library we pass this to annotates this like so:

`     * @param string|DocumentNode $source`

so i am going to leave it like this.